build-tools/packages/build-tools/src/common/monoRepo.ts

```diff
@@ -187,15 +187,17 @@ export class MonoRepo {
 			this.packages.push(Package.load(path.join(pkgDir, "package.json"), kind, this));
 		}
 
+		if (packageManager === "pnpm") {
+			const pnpmWorkspace = path.join(repoPath, "pnpm-workspace.yaml");
+			const workspaceString = readFileSync(pnpmWorkspace, "utf-8");
+			this.workspaceGlobs = YAML.parse(workspaceString).packages;
+		}
+
 		// only needed for bump tools
 		const lernaPath = path.join(repoPath, "lerna.json");
 		if (existsSync(lernaPath)) {
 			const lerna = readJsonSync(lernaPath);
-			if (packageManager === "pnpm") {
-				const pnpmWorkspace = path.join(repoPath, "pnpm-workspace.yaml");
-				const workspaceString = readFileSync(pnpmWorkspace, "utf-8");
-				this.workspaceGlobs = YAML.parse(workspaceString).packages;
```

Comment:
How does workspaceGlobs get set now?

Reply:
Nvm, reviewing on my phone and the new code was scrolled out of view. Now see it.